aws/resource_aws_route_table_test.go

```diff
@@ -286,27 +335,43 @@ func TestAccAWSRouteTable_panicEmptyRoute(t *testing.T) {
 		CheckDestroy:  testAccCheckRouteTableDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config:      testAccRouteTableConfigPanicEmptyRoute,
-				ExpectError: regexp.MustCompile("The request must contain the parameter destinationCidrBlock or destinationIpv6CidrBlock"),
+				Config:      testAccAWSRouteTableConfigPanicEmptyRoute(rName),
+				ExpectError: regexp.MustCompile("The request must contain exactly one of: destinationCidrBlock, destinationIpv6CidrBlock, destinationPrefixListId"),
```

Comment:
This changed with the release of managed prefix lists: https://github.com/terraform-providers/terraform-provider-aws/issues/13986.

Reply:
See #15274.

Reply:
This test passes in GovCloud, but fails in Commercial. I moved validation to the provider in #15261